### DISCUSSION TITLE
Ingestion Pipeline

### DISCUSSION THREAD

**Discussion by CrossBound** [datalust/seq-tickets#1217] [Ideas]
I previously posted this GitHub issue, but sounds like it wasn't on the near future horizon. I'm disappointed that closing it so that no-one gets eyes on it any more is the best way to handle this but I guess it is what it is.

Do you have any idea if this is going to be considered at all in the future?

https://github.com/datalust/seq-tickets/issues/1028

**Comment by nblumhardt** [comment]
Hi David! Thanks for the nudge 👍 

Since we closed the original ticket, GitHub has introduced "Discussions" and we've started using them to track desirable but not-yet-committed features. I went back and converted the original issue so that we can keep discussion going/eyes on it over there:

https://github.com/datalust/seq-tickets/discussions/1218

Will continue the discussion over there! :-)

**Reply by nblumhardt** [reply]
LOL, sorry - and now I realise that I'm replying to a discussion thread and not an issue 😆 .... maybe I should grab another coffeee....
